scipio/src/file_stream.rs

```diff
@@ -682,6 +682,8 @@ impl StreamWriterBuilder {
     ///
     /// [`StreamWriter`]: struct.StreamWriter.html
     pub fn with_buffer_size(mut self, buffer_size: usize) -> Self {
+        let buffer_size = std::cmp::max(buffer_size, 1);
+        self.buffer_size = self.file.align_up(buffer_size as _ ) as usize;
```

Comment:
Same thing here, this is a sane behaviour and we should advertise it in the documentation.